avocado/utils/configure_network.py

```diff
@@ -92,3 +96,84 @@ def unset_ip(interface):
         return True
     except Exception as ex:
         raise NWException("ifdown fails: %s" % ex)
+
+
+class HostInfo:
+    """
+    class for host function
+    """
+
+    def ping_check(self, interface, peer_ip, count, option=None, flood=False):
+        """
+        Checks if the ping to peer works.
+        """
+        cmd = "ping -I %s %s -c %s" % (interface, peer_ip, count)
+        if flood is True:
+            cmd = "%s -f" % cmd
+        elif option is not None:
+            cmd = "%s %s" % (cmd, option)
+        if process.system(cmd, shell=True, verbose=True,
+                          ignore_status=True) != 0:
+            return False
+        return True
+
+    def set_mtu_host(self, interface, mtu):
+        """
+        set mtu size in host interface
+        """
+        cmd = "ip link set %s mtu %s" % (interface, mtu)
+        try:
+            process.system(cmd, shell=True)
+            cmd = "ip add show %s" % interface
+            mtuvalue = process.system_output(cmd, shell=True).decode("utf-8") \
+                                                             .split()[4]
+            if mtuvalue == mtu:
+                return True
+        except Exception as ex:
+            log.error("setting mtu value in host failed: %s", ex)
+        return False
+
+
+class PeerInfo:
+    """
+    class for peer function
+    """
+
+    def __init__(self, host, port=None, peer_user=None,
+                 key=None, peer_password=None):
+        """
+        create a object for acsses remote machnine
+        """
+        self.session = Session(host, port=port, user=peer_user,
+                               key=key, password=peer_password)
+
+    def set_mtu_peer(self, peer_interface, mtu):
+        """
+        set mtu size in peer interface
+        """
+        cmd = "ip link set %s mtu %s" % (peer_interface, mtu)
+        try:
+            self.session.cmd(cmd)
```

Comment:
here as well if command failed in the remote machine it needs to be raise as Exception

Reply:
here command error can't handle because in ssh they dont have any exception class, if needed we handle that later once Exception class adding to ssh file.

Reply:
ok sure